### ISSUE TITLE
Consider using a newer version of "wasm-opt", by default

### ISSUE THREAD

**Issue by andreibancioiu** [multiversx/mx-sdk-py-cli#243]
Newer versions of `rustc` might generate WASM bytecode that isn't properly handled by `wasm-opt` v105.

E.g.

```
rustc 1.70.0-nightly (9df3a39fb 2023-04-11)
```

Wasm-opt error:

```
[wasm-validator error in function 47] unexpected false: all used features should be allowed, on ...
```

Also see: https://t.me/MultiversXDevelopers/111728

**Comment by andreibancioiu** [comment]
Temporary workaround:

```
mxpy config set dependencies.wasm-opt.tag version_112
mxpy deps install wasm-opt --overwrite
```

**Comment by andreibancioiu** [comment]
Fixed by #252.

**Comment by andreibancioiu** [comment]
Also see: https://github.com/multiversx/mx-sdk-py-cli/pull/256
